core/server/common/src/main/java/alluxio/master/BackupManager.java

```diff
@@ -76,6 +75,9 @@
   private long mBackupEntriesCount;
   private long mRestoreEntriesCount;
 
+  private long mBackupTimeMs;
+  private long mRestoreTimeMs;
```

Comment:
Should we set them to `-1` to indicate there was no backup or restore completed?

Reply:
Good idea! Set it to -1